File: documentation/user/en/query/requirements/examples/fetching/referenceContentNested.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeEquals("code", "samsung-galaxy-watch-4")
				),
				require(
					entityFetch(
						attributeContent("code"),
						referenceContent(
							"groups",
							entityFetch(
								attributeContent("code"),
								referenceContent(
									"tags",
									entityFetch(
										attributeContent("code"),
										referenceContent("categories")
									)
								)
							)
						)
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/requirements/examples/fetching/referenceContentNested.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeEquals("code", "samsung-galaxy-watch-4")
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code"),
        			ReferenceContent(
        				"groups",
        				EntityFetch(
        					AttributeContent("code"),
        					ReferenceContent(
        						"tags",
        						EntityFetch(
        							AttributeContent("code"),
        							ReferenceContent("categories")
        						)
        					)
        				)
        			)
        		)
        	)
        )
	)
);